### ISSUE TITLE
posta.com.tr - Anti-Astrology Broke The Website

### ISSUE THREAD

**Issue by bkrucarci** [DandelionSprout/adfilt#536]
Website : https://www.posta.com.tr/
Screenshot with AsTrology Filter **enabled** : https://imgur.com/iNFMEfp.png
Screenshot with AsTrology Filter **disabled** : https://imgur.com/1TE03PV.png

Thanks

**Comment by DandelionSprout** [comment]
I seem to have problems reproducing it. Which web browser are you using, and which adblocker and lists are you using?

**Comment by bkrucarci** [comment]
> I seem to have problems with reproducing it. Which web browser are you using, and which adblocker and lists are you using?

I am using Brave Stable (latest) + AdGuard extension v4.0.125 
Filter : Base + Tracking + Social + Annoyance + Anti-Astrology & Anti-Prayer Time

Same problem also available with Firefox + uBO (same filters)

**Comment by DandelionSprout** [comment]
Okay, I only *now* saw how to reproduce it. The astrology article had vanished by the time I got around to testing it, so I had to test by using `posta.com.tr##a[href*=super-loto-]` instead.

Upon testing this, it appears the feed holder contains `data-column-count="4"`, which cause major problems when subtracting 1 or more articles from the feed, and which I honestly don't think is possible to fix in uBO. So I'm stumped on this.
